lib/attrsets.nix

```diff
@@ -494,4 +494,26 @@ rec {
   zip = builtins.trace
     "lib.zip is deprecated, use lib.zipAttrsWith instead" zipAttrsWith;
 
+  /* Return the cartesian product of attribute set value combinations.
```

Comment:
It would be extra awesome if this documentation would be in the nixpkgs manual.

Reply:
done.